src/main/telemetry/telemetry.h

```diff
@@ -50,6 +50,7 @@ typedef struct telemetryConfig_s {
     uint8_t hottAlarmSoundInterval;
     uint8_t pidValuesAsTelemetry;
     uint8_t report_cell_voltage;
+    uint8_t flysky_sensors[15];
```

Comment:
Instead of using a 'magic number', use a define like `IBUS_SENSOR_COUNT` for the number of supported sensors. And then reuse it in places like `settings.c`.